Monika-After-Story/MonikaModDev
# Default selectors to seen

This PR adds a new param to addEvent, `defaultSeenever`. It's the equivalent of adding a `persistent._seen_ever["topic_evl"] = True` after its addEvent.

I have also updated the documentation on the addEvent function.

Additionally, selectors now all default to seen with the exception of the ribbon selector, which is now `aff_range`d (Normal+), to indicate to the user that the Appearance tab is something where they can go to change Moni's appearance.

# Testing:
- Verify BRBs don't show up in unseen still,
- Verify that the happy birthday topics in the compliments menu are not italicized if you haven't used them before
- Verify that all selectors aside from the ribbon selector will default to a True _seen_ever
